src/queryResult/queryResultWebViewController.ts

```diff
@@ -93,7 +94,14 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
                     message.numberOfRows,
                 );
             let currentState = this.getQueryResultState(message.uri);
-            if (currentState.isExecutionPlan) {
+            if (
```

Comment:
Small readability thing, but if the bulk of the logic is inside the condition, then it's nice to invert it and have it bail out early.  That way, all the long longer logic isn't indented as deeply.

instead of
```
if (something) {
  doStuff()
  doStuff()
  doStuff()
  doStuff()
  doStuff()
  doStuff()
  doStuff()
}
return state;
```

it's easier to read if it's:
```
if (!something) {
return state; // short null/bail case is first
}

doStuff()
doStuff()
doStuff()
doStuff()
doStuff()
doStuff()
doStuff()
return state;
```